mqtt/mqtt-edgehub/src/auth/local.rs

```diff
@@ -0,0 +1,73 @@
+use std::error::Error as StdError;
+
+use async_trait::async_trait;
+
+use mqtt_broker_core::auth::{AuthId, AuthenticationContext, Authenticator};
+
+#[derive(Debug, Default)]
+pub struct LocalAuthenticator;
+
+impl LocalAuthenticator {
+    pub fn new() -> Self {
+        Self::default()
+    }
+}
+
+#[async_trait]
+impl Authenticator for LocalAuthenticator {
```

Comment:
can you please add some comments on why we need this LocalAuthenticator?

Reply:
There is a comment when we actually use this authenticator.